pkg/terror/error_list.go

```diff
@@ -392,7 +392,8 @@ const (
 	codeSyncerFailpoint
 	codeSyncerReplaceEvent
 	codeSyncerOperatorNotExist
-	codeSyncerReplcaeEventNotExist
+	codeSyncerReplaceEventNotExist
+	codeSyncerUnitNewTracker
```

Comment:
How about move it to `Schema-tracker error code`

Reply:
https://github.com/pingcap/dm/pull/1032/commits/8fc1466a40ed0df848a833d2c7c6df63270b7d4b